contrib/seeds/README.md

```diff
@@ -4,7 +4,9 @@ Utility to generate the seeds.txt list that is compiled into the client
 (see [src/chainparamsseeds.h](/src/chainparamsseeds.h) and other utilities in [contrib/seeds](/contrib/seeds)).
```

Comment:
In commit "Use GetDesireableServiceFlags in static seeds, document this."

I'm probably just tired, but I found this commit message confusing. Maybe say:

> broke inserting entries into addrman from static seeds (as well as dnsseeds which did not support service bits)

instead of:

> broke inserting entries into addrman from dnsseeds which did not support service bits, as well as static seeds

since static seeds are really what's being addressed in this commit.